resources/js/common/components/AchievementsListItem/AchievementDateMeta/AchievementDateMeta.tsx

```diff
@@ -70,6 +73,18 @@ export const AchievementDateMeta: FC<AchievementDateMetaProps> = ({
         </p>
       ) : null}
 
+      {isDemoted ? (
+        <span
+          className="flex cursor-help items-center"
```

Comment:
UX: Recommend also adding `gap-1` or `gap-1.5` to add some space between the icon and the label, whichever of the two looks better.

Reply:
That's what the `ml-0.5` was supposed to be doing. Replaced with `gap-1`.